src/app/modules/main/chat_section/module.nim

```diff
@@ -358,7 +371,7 @@ method onChatUnmuted*(self: Module, chatId: string) =
   self.view.chatsModel().muteUnmuteItemOrSubItemById(chatId, false)
 
 method onMarkAllMessagesRead*(self: Module, chatId: string) =
-  self.view.chatsModel().setHasUnreadMessage(chatId, value=false)
+  self.updateNotifications(chatId, 0, 0)
```

Comment:
Can you put the arg name so we can know what `0` means here. Or maybe we could use a default argument in the function.